backend/core/views.py

```
@@ -2604,8 +2604,8 @@ def get_mapped_ids(
                 )
 
             case "riskassessment":
-                _fields["project"] = Project.objects.get(
-                    id=link_dump_database_ids.get(_fields["project"])
+                _fields["perimeter"] = Perimeter.objects.get(
```

Ruff (0.8.2)

`Perimeter` may be undefined, or defined from star imports

(F405)